mlflow/server/handlers.py

```diff
@@ -443,6 +450,28 @@ def _log_batch():
     return response
 
 
+@catch_mlflow_exception
+def _log_model():
+    request_message = _get_request_message(LogModel())
+    try:
+        _logger.error(request_message.model_json)
+        model = json.loads(request_message.model_json)
+    except JSONDecodeError:
+        raise MlflowException("Model info is not a valid json.", error_code=INVALID_PARAMETER_VALUE)
```

Comment:
```suggestion
        raise MlflowException("Model info '%s' is not a valid json." % request_message.model_json, error_code=INVALID_PARAMETER_VALUE)
```

Include the bad JSON in the error message for ease of debugging?

Reply:
Makes sense.